tests/integration/test_events.py

```diff
@@ -1596,3 +1596,22 @@ def cleanup(
                     logGroupName=log_group_name, logStreamName=log_stream["logStreamName"]
                 )
             logs_client.delete_log_group(logGroupName=log_group_name)
+
+    def test_put_target_id_validation(self, sqs_create_queue, events_client, sqs_client, snapshot):
```

Comment:
```suggestion
    @pytest.mark.aws_validated
    def test_put_target_id_validation(self, sqs_create_queue, events_client, sqs_client, snapshot):
```

I guess this was executed against AWS since it has a snapshot right?